### PR TITLE
Added a PaymentTypeViewModel with all the correct properties

### PR DESCRIPTION
**Pull request by Jgregnier**
## Pull Request Description

This view model will be used with Debbie's new payment form. 
With this view model being given to her form the inputs should now conform to the properties and restrictions of the PaymentType model. 
## Steps to Test
1. Check for spelling errors.
2. Check for any other errors
## Link to Feature Ticket
#112

### CODE REVIEW

**Review by JamsDuke — COMMENTED** [review]
Looks solid, good to merge.

File: ViewModels/PaymentTypeViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using BangazonWeb.Models;
+using BangazonWeb.Data;
+
+namespace BangazonWeb.ViewModels
+{
+    //Class Name: PAymentTypeViewModel
```
Comment (delaineackerman) [inline ViewModels/PaymentTypeViewModel.cs]: lowercase a, not uppercase

**Comment by HubDub** [comment]
Looks good to me.

**Review by csmalley89 — COMMENTED** [review]
👍

**Review by delaineackerman — APPROVED** [review]

**Pull request merged**